README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img alt="Files Logo" src="src/Files.App%20(Package)/Assets/AppTiles/Release/StoreLogo.scale-400.png" width="100px" />
+  <img alt="Files Logo" src="https://raw.githubusercontent.com/files-community/Website/main/static/branding/logo-dark.svg" width="100px" />
```

Comment:
It's not recommended to add a dependency on the website repo so I think this change should be reverted.